tests/integration/regenerate.sh

```diff
@@ -141,8 +141,11 @@ for ((i = 0; i < num_of_tests; i++)); do
       set +e
     fi
 
-    run_test
-    TEST_STATUS=$?
+    TEST_STATUS=1
+    if [ -z $SKIP_TEST ]; then
```

Comment:
What's the purpose of this? To save time?

Reply:
I actually forget why I needed this now...I think a test was "passing" but I still wanted to regenerate.

Reply:
that's my guess, but this won't do what you want, unless you also use `FORCE_USE_LLM`.

Reply:
Okay I guess you were using both `SKIP_TEST` and `FORCE_USE_LLM`. That would make sense, as it will force using the real LLM.